matscipy/hessian_finite_differences.py

```diff
@@ -21,11 +21,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import numpy as np
+"""Deprecated module."""
 
-from scipy.sparse import coo_matrix
+from numpy import deprecate
+from .numerical import numerical_hessian
 
 
+@deprecate(new_name="numerical.numerical_hessian")
```

Comment:
Do we need this deprecation or can we just remove it?

Reply:
It's not very nice to users to break API with no warning. That concern aside, I haven't changed all the tests to use the new/renamed methods. But pytest reports deprecation warnings, so it's just a matter of methodically going through a list.

Reply:
Sounds good, let's leave with the deprecation warning.